plugins/node/instrumentation-undici/test/undici.test.ts

```diff
@@ -207,7 +210,13 @@ describe('UndiciInstrumentation `undici` tests', function () {
       assert.ok(spans.length === 0, 'ignoreRequestHook is filtering requests');
     });
 
-    it('should create valid spans for different request methods', async function () {
+    it.only('should create valid spans for different request methods', async function () {
+      if (semver.lt(UNDICI_VERSION, '5.12.0')) {
```

Comment:
if we drop support for `<5.12.0` this won't be necessary. Neither the try/catch of this test